test/fixtures/metadata/exercises/fish/.deprecated

```diff
@@ -0,0 +1 @@
+fish
```

Comment:
The `.deprecated` file should be empty. It would typically be created with `touch exercises/fish/.deprecated`.